docs/spark/how-to-guides/hdinsight-deploy-methods.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Submit a .NET for Apache Spark job to Azure HDInsight
+description: Learn how to submit a .NET for Apache Spark job to Azure HDInsight using spark-submit and Apache Livy.
+ms.date: 11/14/2019
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+# Submit a .NET for Apache Spark job to Azure HDInsight
+
+There are two ways to deploy your .NET for Apache Spark job to HDInsight: `spark-submit` and Apache Livy.
+
+## Deploy using spark-submit
+
+You can use the [spark-submit](https://spark.apache.org/docs/latest/submitting-applications.html) command to submit .NET for Apache Spark jobs to Azure HDInsight.
+ 
+1. Navigate to your HDInsight Spark cluster in Azure portal, and then select **SSH + Cluster login**.
+
+2. Copy the ssh login information and paste the login into a terminal. Sign in to your cluster using the password you set during cluster creation. You should see messages welcoming you to Ubuntu and Spark.
+
+3. Use the **spark-submit** command to run your app on your HDInsight cluster. Remember to replace **mycontainer** and **mystorageaccount** in the example script with the actual names of your blob container and storage account.
+
+   ```bash
+   $SPARK_HOME/bin/spark-submit \
+   --master yarn \
+   --class org.apache.spark.deploy.DotnetRunner \
+   wasbs://mycontainer@mystorageaccount.blob.core.windows.net/microsoft-spark-2.3.x-0.6.0.jar \
```

Comment:
We may want to add a note telling users to adjust `2.3.x-0.6.0` based on their specific version of Spark and .NET for Apache Spark, i.e.:

Note: Replace `microsoft-spark-2.3.x-0.6.0.jar` with the appropriate jar file you're using for deployment. `2.3.x` represents the version of Apache Spark, and `0.6.0` represents the version of the [.NET for Apache Spark worker](https://github.com/dotnet/spark/releases).